onnxmltools/convert/sklearn/operator_converters/CalibratedClassifierCV.py

```diff
@@ -0,0 +1,168 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._apply_operation import apply_add, apply_cast, apply_div, apply_exp, apply_mul, apply_reshape
+from ...common._registration import get_converter, register_converter
+from .._parse import sklearn_operator_name_map, _get_sklearn_operator_name 
+import numpy as np
+from sklearn.naive_bayes import BernoulliNB
+from sklearn.naive_bayes import MultinomialNB
+
+class Oper:
+    def __init__(self, model, inputs, op_type):
+        self.raw_operator = model
+        self.inputs = inputs
+        self.type = op_type
+
+def convert_sklearn_calibrated_classifier_cv(scope, operator, container, flag=False, model=None):
```

Comment:
add some words about what's the flag?

Reply:
This flag controls the behavior of converter functions when producing outputs. If it's true, the converter function may return a string (array) which stands for its output(s). However, this flag somehow breaks the design for separating converter and parser. My suggestion is to move anything relating to parsing scikit-learn objects to `_parse.py` as suggested in another PR.